aspnet/mvc/views/tag-helpers/authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImports.cshtml

```diff
@@ -1,11 +1,11 @@
 ﻿@using AuthoringTagHelpers
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
-@addTagHelper "AuthoringTagHelpers.TagHelpers3.EmailTagHelper, AuthoringTagHelpers"
+@addTagHelper "AuthoringTagHelpers.TagHelpers.EmailTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.WebsiteInformationTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.ConditionTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.AutoLinkerHttpTagHelper, AuthoringTagHelpers"
 @addTagHelper "AuthoringTagHelpers.TagHelpers.AutoLinkerWwwTagHelper, AuthoringTagHelpers"
-@addTagHelper "AuthoringTagHelpers.TagHelpers3.BoldTagHelper, AuthoringTagHelpers"
+@addTagHelper "AuthoringTagHelpers.TagHelpers.BoldTagHelper, AuthoringTagHelpers"
```

Comment:
Atleast this line https://github.com/aspnet/Docs/blob/dev/aspnet/mvc/views/tag-helpers/authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/_ViewImports.cshtml#L3 needs to change. It is being showcased here https://docs.asp.net/en/latest/mvc/views/tag-helpers/intro.html and having the namespace as `TagHelpers3` is confusing.